core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -289,8 +289,27 @@ public static Boolean isFusion(String variant) {
         return flag;
     }
 
+    public static Alteration getRevertFusions(Alteration alteration, Set<Alteration> fullAlterations) {
+        if (fullAlterations == null) {
+            return getRevertFusions(alteration);
+        } else {
+            String revertFusionAltStr = getRevertFusionName(alteration);
```

Comment:
Could you explain the logic in here a little bit? What does match.isPresent() mean?

Reply:
@victoria34 this is a new method in Java 8, https://www.baeldung.com/java-optional. In here, I do a filter through stream, it produces an optional. If there is a match, then the isPresent is true.